### PR TITLE
[Refactor] Specify mmengine version

### PR DESCRIPTION
**Pull request by Ben-Louis**
## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: requirements/poseval.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 poseval@git+https://github.com/svenkreiss/poseval.git
+shapely==1.8.4
```
Comment (ly015) [inline requirements/poseval.txt:2]: Is `shapely` required by `poseval`?

Comment (Ben-Louis) [reply requirements/poseval.txt:2]: yes

Comment (ly015) [reply requirements/poseval.txt:2]: It seems already in the requirements of `poseval` (https://github.com/svenkreiss/poseval/blob/master/setup.py#L15). Won't it be automatically installed?

Comment (Ben-Louis) [reply requirements/poseval.txt:2]: There is a new release of `shapely` today (v1.8.5), which will be automatically installed. This version of `shapely` seems incompatible with the unittest environment, and will make CI fail.

**Review by ly015 — APPROVED** [review]

**Pull request merged**